### DISCUSSION TITLE
The difference between score and total

### DISCUSSION THREAD

**Discussion by imcvampire** [jonhue/acts_as_favoritor#133] [Q&A] [answered]
Hi,

First of all, I want to thank you for your open source contribution.

I want to know the difference between `favoritable_score` and `favoritable_total` on cache

Thanks

**Comment by jonhue** [comment, marked as answer]
The total counts all favorites that were recorded, while the score factors in favorites that were removed. In most use cases the score is the most useful.

**Comment by KenneyE** [comment]
Thanks @jonhue. I think adding that exact sentence to the README would worthwhile.

**Reply by jonhue** [reply]
done!
